engine/rpcserver.go

```diff
@@ -1021,7 +1021,23 @@ func (s *RPCServer) CancelOrder(_ context.Context, r *gctrpc.CancelOrderRequest)
 
 // CancelAllOrders cancels all orders, filterable by exchange
 func (s *RPCServer) CancelAllOrders(_ context.Context, r *gctrpc.CancelAllOrdersRequest) (*gctrpc.CancelAllOrdersResponse, error) {
-	return &gctrpc.CancelAllOrdersResponse{}, common.ErrNotYetImplemented
+	if r.Exchange != "kraken" { // at the moment implemented only on the Kraken
+		return &gctrpc.CancelAllOrdersResponse{}, common.ErrNotYetImplemented
+	}
+
+	exch := s.GetExchangeByName(r.Exchange)
+	if exch == nil {
+		return &gctrpc.CancelAllOrdersResponse{}, errExchangeNotLoaded
+	}
+
+	resp, err := exch.CancelAllOrders(nil)
```

Comment:
Some exchanges like Bithumb, BTSE utilise order.Cancel to retrieve open orders depending on order-side and currencies. This needs to be reflected and options added to the RPC and GCTCLI.